core/modules/system/config/system.core.json

```
@@ -80,6 +80,13 @@
     "user_mail_status_canceled_notify": 0,
     "user_register": "admin_only",
     "user_signatures": 0,
+    "user_password_constraints": {
+        "constraints_active": "1",
```

@herbdool related to your other comment in this PR, this should also be numeric:

```suggestion
        "constraints_active": 1,
```

A problem that has been bothering me for some time though is that we seem to have a mix of numeric values `1` and `0`, as well as strings `"1"` or `"0"` in config for things that should be on/off true/false. Same with weight fields too I believe, where the numeric value of the weight is saved as a string. I have already raised a DX issue about that here, which I would like to see sorted at some point: https://github.com/backdrop/backdrop-issues/issues/2557